pkg/config/config.go

```diff
@@ -32,7 +39,44 @@ func Get() Config {
 		},
 		StaticFilesDir: getEnv("STATIC_FILES_DIR", ""),
 		ApiRoutePrefix: "/api", //should start with a "/",
+		DB: DBConfig{
+			ConnectionString: getDBConnectionString(),
+			MaxConnections:   getEnvInt("DB_MAX_CONNECTIONS", 100),
+		},
+	}
+}
+
+// builds a libpq compatible connection string e.g. "user=postgresql host=localhost password=postgres
+func getDBConnectionString() string {
+
+	// These are all of the options supported by pq
+	dbConnectionConfig := map[string]string{
+		"dbname":                    getEnv("PGDATABASE", "mobile_security_service"),
+		"user":                      getEnv("PGUSER", "postgresql"),
+		"password":                  getEnv("PGPASSWORD", "postgres"),
+		"host":                      getEnv("PGHOST", "localhost"),
+		"port":                      getEnv("PGPORT", "5432"),
```

Comment:
Should it not be env properties? Will ii always be a HOST=localhost?

Reply:
These are env properties. localhost is the default value but they can be overridden with environment variables. Refer to the [documentation](https://github.com/aerogear/mobile-security-service#add-your-own-env-file) on using your environment variables.